src/main/java/org/reaktivity/k3po/nukleus/ext/internal/behavior/NukleusStreamFactory.java

```diff
@@ -110,6 +111,7 @@ private void onBegin(
             BeginFW begin)
         {
             final long streamId = begin.streamId();
+            authorization = begin.authorization();
```

Comment:
Store `authorization` on the `channel` instead, like `streamId` ?

Reply:
done